### DISCUSSION TITLE
AMD 6900XT support

### DISCUSSION THREAD

**Discussion by mrsikboy** [Rem0o/FanControl.Releases#596] [General]
hi guys, ive only found 1 post regarding amd cards and reply was its hard to get amd cards working.

Is there a work around for this. Any help is appreciated

I have a Asrock 6900XT and you can select the card from  the startup but when the complete auto setup is  done, its asking for a rpm sensor is required

![1](https://user-images.githubusercontent.com/18359769/164843142-c7b0be10-b0d6-4ae8-a0ac-a18d5038ee40.jpg)
![2](https://user-images.githubusercontent.com/18359769/164844096-bb4301f7-b1fc-4fed-a407-76d7f9fceabb.jpg)

**Comment by Rem0o** [comment]
Please see the README.

There is no reliable and available way to control AMD cards on windows right now that works with all modern cards.

**Comment by mrsikboy** [comment]
i was just confused, with the base software that is used, recognises the card and all its sensors

**Comment by mrsikboy** [comment]
LHM detects card ok, thats where im confused

![2](https://user-images.githubusercontent.com/18359769/164872975-85f02c02-7c9f-41bd-9345-f76e4dd1637a.jpg)

**Comment by SLCarlsson** [comment]
Yeah, but it seems that you cannot take control of the fans via LHM.

**Comment by r-tango** [comment]
You have gotten further than some of us. I would be stoked if I could get the temp sensor on my 6700XT to be picked up. A few others posted in issues that nothing showed with 6800XT's as well. Any trick to how you did that?

**Reply by mrsikboy** [reply]
are you refering to me?   if so, did nothing, just installed libre hardware monitor and it shows up

**Reply by mrsikboy** [reply]
argus monitor is what im using, but dont want to keep paying. prefer to donate to this software

can control everything with argus

**Reply by r-tango** [reply]
Yes sir, I haven't tried libre on its own specifically. I will give that a go and see if it helps. As soon as I got this to work on my other pc, I hit that donate button, as it worked flawlessly on that machine. Even though it only partially works on this one it is still worth it.

**Reply by mrsikboy** [reply]
so you can control your 6800xt? with fancontrol

**Reply by r-tango** [reply]
not really, just need to see the sensor so i can control the case fans based on the GPU temps on a mix curve

**Reply by SLCarlsson** [reply]
> not really, just need to see the sensor so i can control the case fans based on the GPU temps on a mix curve

That is also my goal with fancontrol. How did you make it work?
Did you install a plugin?

**Reply by r-tango** [reply]
On the RX480, hero vii, 4790K system all I did was install fancontrol, follow the quick start prompts and it just worked.

On the 6700xt, X570I, 5800X system, it only controls the case and CPU cooler fans, and sees the temps on the MB, nvme's and CPU. Nothing from either the power supply or GPU.

Working on the RGB today and will start working on the fans (again) shortly after. But I am a total amateur, so don't expect much, haha.

**Reply by r-tango** [reply]
Alright, got to see the GPU temp now at least on the 6700xt. Was so excited I had to post it right away. But the short of it is, I had an issue with a new nvme drive and after all the trouble shooting, the GPU temp showed up. Not that I enjoyed the long ass process, and the nvme still doesn't work, but what I think did it for fancontrol was either when I cleared the CMOS or ran with CSM enabled through the BIOS. Going to set up fancontrol now and see if the fans and such are also all on there now. 
![Screenshot 2022-04-29 200845](https://user-images.githubusercontent.com/104403369/166086443-5bd562bb-213a-4b47-884e-9a9210162252.png)

**Reply by mrsikboy** [reply]
so you could not see the 6700xt in that drop down previously?

what is csm in the bios. not sure what that would change

**Reply by r-tango** [reply]
not at all.
it's a compatibility mode that allows legacy devices on a Asus Rog Strix x570I

**Reply by r-tango** [reply]
idk if these two are entirely related, but i can tell you when adrenalin is active or installed it is blocking fancontrol from seeing the GPU temps and controlling the fans.
![Screenshot 2022-04-30 101236](https://user-images.githubusercontent.com/104403369/166113544-93e3d833-f7e4-42fb-9433-fdd378e3418d.png)

**Reply by r-tango** [reply]
Had to do a fresh install of Adrenalin after fancontrol was all set up as well.

**Comment by mrsikboy** [comment]
need to control the gpu fans, so amd software is not used as its crap, using the gpu sensor to control case fans etc is ok, but 1 solution for everything is preferred. only argus monitor that i use does it all.

**Reply by r-tango** [reply]
I can see it but have no control over the gpu fan speeds. Sitting at zero rn and when I hit manual to 100 it doesn't move off zero but shows it accurately in the app. Seems like control is locked to AMD's Adrenalin app.

**Comment by r-tango** [comment]
I only got it to work entirely on my older system. That has a rx480
sapphire on it with a hero vii, 4790K and some Corsair and noctua fans. And
it is awesome how well it does mind you. But I have had no luck on the
other newer amd system with the 6700xt, rog x570i, 5800x and a mix of tt
fans, a noctua and a cooler master fan. Everything on the MSI 6700xt OC is
nonexistent. When I have the time I'm going to try a clean install with
libre first and see if it makes a difference.
